docs/configuration.md

```diff
@@ -0,0 +1,140 @@
+## How to configure RIF Services for local development.
+
+When you run RConnect on Regtest with a local node you need to setup all the 
+neccesary infrastructure to allow the wallet to work with RIF Services. 
+To do so you need to configure some information about the API's that allow
+us to access those services.
+
+If we see the RConnect General Architecture Diagram:
+
+![General Diagram](./general-rconnect-architecture.png)
```

Comment:
it makes sense to have this here, but what doesn't seem so logical to me is that:
1. we also have this diagram on the main `readme.md` file
2. we have an even _more_ specific diagram there as well

i would maybe just have both here and link them from there